document.go

```diff
@@ -27,6 +27,9 @@ type Document interface {
 	NumPlainTextBytes() uint64
 
 	AddIDField()
+
+	SetStoredFieldsBytes(uint64)
```

Comment:
This doesn't look right, we shouldn't need this method.

Reply:
Done.